impl.go

```diff
@@ -447,7 +459,19 @@ to prevent shell globbing.
 		}
 	}
 
-	fns, err := funcs(iface, *flagSrcDir, EmitComments(*flagComments))
+	if *flagRecvPkg == "" {
+		//  "   s *Struct   " , receiver: Struct
+		recvs := strings.Split(strings.TrimSpace(recv), " ") // [0]:s [1]:*Struct
+		receiver := strings.TrimPrefix(recvs[len(recvs)-1], "*")
+		pkg, _, err := typeSpec("", receiver, *flagSrcDir)
+		if err == nil {
+			*flagRecvPkg = pkg.Package.Name
+		} else {
+			*flagRecvPkg = "---" // Cannot be matched with any package
```

Comment:
Why is `"---"` better than `""`?

Reply:
I just think maybe pkg.Package.Name is "" (maybe that's not possible). So I don't use the default value of string.

Reply:
I don't think that's possible. Let's not worry about that possibility.

Reply:
OK.